x/group/server/testsuite/genesis.go

```diff
@@ -77,7 +78,14 @@ func (s *IntegrationTestSuite) TestInitExportGenesis() {
 	genesisBytes, err := cdc.MarshalJSON(genesisState)
 	require.NoError(err)
 
-	genesisData := map[string]json.RawMessage{group.ModuleName: genesisBytes}
+	ecocreditGenesisState := ecocredit.DefaultGenesisState()
+	ecocreditGenesisBytes, err := cdc.MarshalJSON(ecocreditGenesisState)
+	require.NoError(err)
+
+	genesisData := map[string]json.RawMessage{
+		group.ModuleName:     genesisBytes,
+		ecocredit.ModuleName: ecocreditGenesisBytes,
+	}
```

Comment:
I'm not so fond of having to put ecocredit module related logic in the group module tests but I guess we have no other choice with the current setup, unless we create public functions `InitGenesis` and `ExportGenesis` that could be tested individually without the server (they would be used in `serverImpl.InitGenesis` and `serverImpl.ExportGenesis`) similarly to what we have with the invariants tests. But then that would force us to set up all the tables again in the tests while this is already handled by the integration test suite here so this is not ideal either. So let's keep it as is for now until we come to a better solution in terms of module wiring / test setup.